src/Block/Service/AbstractBlockService.php

```diff
@@ -70,7 +98,18 @@ public function __construct($templatingOrDeprecatedName = null, EngineInterface
      */
     public function renderResponse($view, array $parameters = [], Response $response = null)
     {
-        return $this->getTemplating()->renderResponse($view, $parameters, $response);
+        if (!$this->twig) {
```

Comment:
```suggestion
        if (null !== $this->twig) {
```

Reply:
This check requires exactly the opposite.
`$this->twig` will be lossely evaluated to `false` if an instance of `Environment` is not set. There is already a check at `__construct()` that prevents this property to hold other value than `Environment`.

Reply:
Oh sorry, I meant `null === $this->twig`.

I prefer explicit type checks as long as we don't have typed properties

Reply:
The explicit type check is emulated with [this check](https://github.com/sonata-project/SonataBlockBundle/pull/621/files#diff-039b4f28b4329f773f4b0fbfda05ea05R80). Regardless that, I've updated this condition as you have proposed.